docs/howto/account-billing/delete-account.md

```diff
@@ -17,6 +17,20 @@ Choose a reason for leaving and type in your current password. Click the red but
 
 ![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)
 
+
+## Using the {{legal_docs.termination_form.name}}
+
+You may be unable to use the standard account deletion procedure using the {{gui}}, because you have lost your administrative credentials or you cannot access your account owner's email address.
+
+In this case, you can instead terminate your account by filling out the [{{legal_docs.termination_form.name}}]({{legal_docs.termination_form.url}}).
+Be sure to add your own signature (for personal accounts), or that of an authorized signatory (for company accounts).
+
+Submit the form via our [{{support}}](https://{{support_domain}}/servicedesk).
+If you have lost your {{support}} credentials, you may also send it by email to [{{support_email}}](mailto:{{support_email}}).
+
+
+## What happens after you delete your account
+
 Three (3) days after sending the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.
```

Comment:
I believe we are on the safe(r) side with the following ever-so-slight rephrasing:
```suggestion
Three (3) days after receiving the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.
```

Reply:
Fair!